CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [Unreleased]
 - Add `forwardErrors` option to pass 404s and other HTTP errors down to the next Express error-handling middleware.
 - Recommend `npm` instead of `yarn` and switch to `npm` lock file.
+- Check file existence asynchronously so we don't block the event loop as much.
```

Comment:
Remove 'we'? "so the event loop is not blocked."

Reply:
Done.